views/components/code-sample/_code-sample.scss

```diff
@@ -1,4 +1,6 @@
 .code-sample {
+	$border-color: #AAA;
```

Comment:
actual color value TBD; is there an established palette we could choose from?

Reply:
There is not currently an established palette. I've been working off the colors from the logo, but those are currently hard coded in the CSS. Not ideal for the long-term, but I would refactor them out in a separate step.

Reply:
Makes sense; being pragmatic is a virtue right now.

If you think this is a sensible change (feel free to reject it), care to choose a suitable color?

Actually, since I've gotten rid of duplicate declarations for this, we don't actually need a variable anymore? Might as well specify the color value directly below.